apollo-api/api/apollo-api.api

```diff
@@ -1120,8 +1120,8 @@ public final class com/apollographql/apollo3/exception/AutoPersistedQueriesNotSu
 
 public final class com/apollographql/apollo3/exception/CacheMissException : com/apollographql/apollo3/exception/ApolloException {
 	public static final field Companion Lcom/apollographql/apollo3/exception/CacheMissException$Companion;
-	public fun <init> (Ljava/lang/String;Ljava/lang/String;)V
-	public synthetic fun <init> (Ljava/lang/String;Ljava/lang/String;ILkotlin/jvm/internal/DefaultConstructorMarker;)V
+	public synthetic fun <init> (Ljava/lang/String;Ljava/lang/String;Ljava/lang/Long;ILkotlin/jvm/internal/DefaultConstructorMarker;)V
+	public final fun getAge ()Ljava/lang/Long;
```

Comment:
This is a false positive of https://github.com/Kotlin/binary-compatibility-validator/issues/36. This API shouldn't be public just yet